src/app/side-bar/dashboard/dashboard.component.html

```diff
@@ -24,7 +24,7 @@ <h5 class="dashboard-container__card-title-detail ml-4">How to use the portal</h
             Indicator Content
           </ng-template>
         </p-image>
-        <div fxLayout="column" class="dashboard-container__card-img-caption">
+        <div fxLayout="column" class="dashboard-container__card-img-caption p-12 p-0 p-12 p-4">
```

Comment:
what are you trying to achieve here by applying all this padding here????

Reply:
@tsehringyangden, please come to me i think u got confused about using p or mixins

Reply:
Resolved.............Thank you